pkg/health/condition/check_backup_ready.go

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package condition
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	druidv1alpha1 "github.com/gardener/etcd-druid/api/v1alpha1"
+	coordinationv1 "k8s.io/api/coordination/v1"
+
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type backupReadyCheck struct {
+	cl client.Client
+}
+
+const (
+	FullBackupSucceeded string = "FullBackupSucceeded"
+	IncrBackupSucceeded string = "IncrementalBackupSucceeded"
+	BackupFailed        string = "BackupFailed"
+	Unknown             string = "Unknown"
+)
+
+func (a *backupReadyCheck) Check(etcd druidv1alpha1.Etcd) Result {
+	//Default case
+	result := &result{
+		conType: druidv1alpha1.ConditionTypeBackupReady,
+		status:  druidv1alpha1.ConditionFalse,
```

Comment:
```suggestion
		status:  druidv1alpha1.ConditionUnknown,
```

We should only set it to `druidv1alpha1.ConditionFalse` after a configurable amount of time has been elapsed.

Reply:
Done